docs/src/main/sphinx/release/release-403.md

```diff
@@ -0,0 +1,72 @@
+# Release 403 (15 Nov 2022)
+
+## General
+
+* Add the amount of data read from external sources during table scans to
+  `EXPLAIN ANALYZE`. ({issue}`14907`)
+* Improve performance of worker-to-worker data transfer encryption when
+  fault-tolerant execution is enabled. ({issue}`14941`)
+* Improve performance of aggregations when input data does not contain nulls. ({issue}`14567`)
+* Fix potential failure when clients do not support variable precision temporal
+  types. ({issue}`14950`)
+* Fix query deadlock in multi-join queries where broadcast join size is
+  underestimated. ({issue}`14948`)
+* Fix incorrect results when `min(x, n)` or `max(x, n)` is used as a window
+  function. ({issue}`14886`)
+* Fix failure for certain queries involving joins over partitioned tables. ({issue}`14317`)
+* Fix incorrect order of parameters in `DESCRIBE INPUT` when they appear in a
+  `WITH` clause. ({issue}`14738`)
+* Fix failure for queries involving BETWEEN predicates over `varchar` columns
+  that contain temporal data. ({issue}`14954`)
+
+## Security
+
+* Allow access token passthrough when using OAuth 2.0 authentication with
+  refresh tokens enabled. ({issue}`14949`)
+
+## BigQuery connector
+
+* Improve performance of `SHOW SCHEMAS` by adding a metadata cache. This can be
+  configured with the `bigquery.metadata.cache-ttl` catalog property, which is
+  disabled by default. ({issue}`14729`)
+* Fix failure when a [row access 
```

Comment:
Don't break label across lines. It's ok it the line extends beyond the soft margin